examples/README.md

```diff
@@ -1,31 +1,39 @@
-# Functions Framework Examples
+# Testing the Functions Framework Examples
 
-This directory contains a series of examples showing how to use the Functions Framework.
+These notes describe how build and tests these examples when developing the Functions Framework for C++. The main
```

Comment:
s/how build and tests/how to build and test/

Reply:
Done.